api/pom.xml

```diff
@@ -18,6 +18,12 @@
 			<version>1.5.8</version>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+				<groupId>org.openmrs.module</groupId>
+				<artifactId>emrapi-condition-list</artifactId>
+				<version>${emrApiVersion}</version>
+				<scope>test</scope>
+		</dependency> 
```

Comment:
Can you clarify what is needed here?  Why do we need to include emrapi-condition-list?  Is this for the hibernate mapping file referenced above?  Is the condition list functionality in the emrapi module or is it in core?

Reply:
@mseaton ,sorry this was a mistake. The issue is i innitially  used the HQLQuery builder ,so i was innitially making a reference of the Condition class in the HQLQuery builder ,so i needed that dependence.
 However when i switched to Using the SQLQuery builder , i had to remove that dependence